model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
```

Comment:
```suggestion
    unit: "{span}"
```
This is technically equivalent, but it is nice to be specific when we can.  Applies here, and to other metrics counting spans below.

Reply:
Fixed in 26e21b2b956d